unittests/Feature/FeatureModelParser.cpp

```diff
@@ -53,6 +53,75 @@ TEST(FeatureModelParser, outOfOrder) {
   EXPECT_TRUE(P.buildFeatureModel());
 }
 
+// TODO: Detect error while parsing
+//
+//  TEST(FeatureModelParser, errorMismatchParentChild) {
+//   auto FS = llvm::MemoryBuffer::getFileAsStream(
+//       getTestResource("error_mismatch_parent_child.xml"));
+//
+//   ASSERT_TRUE(FS);
+//
+//   auto P = FeatureModelXmlParser(FS.get()->getBuffer().str());
+//
+//   EXPECT_TRUE(P.verifyFeatureModel());
+//   EXPECT_FALSE(P.buildFeatureModel());
+// }
+
+TEST(FeatureModelParser, errorMissingChild) {
```

Comment:
For the tests in general, are they ensuring enough? It looks a bit strange that we only verify/build an then accept. It could be that the tests check to general maybe, we should talk about this.